src/app/feedback/feedback.component.ts

```diff
@@ -95,12 +96,14 @@ export class FeedbackComponent implements OnInit, AfterViewInit, OnDestroy {
 
   getFeedback() {
     const selector = !this.user.isUserAdmin ? { 'owner': this.user.name } : { '_id': { '$gt': null } };
-    this.couchService.findAll(this.dbName, findDocuments(selector, 0, [ { 'openTime': 'desc' } ]))
-      .subscribe((data) => {
-        this.feedback.data = data;
+    forkJoin([
+      this.couchService.findAll('_users'),
```

Comment:
Please use `usersListener` so the documents have the `fullName` property for the template.